kpi/tests/api/test_api_environment.py

```diff
@@ -100,3 +106,45 @@ def test_template_context_processor(self):
         template = Template('{{ config.TERMS_OF_SERVICE_URL }}')
         result = template.render(context)
         self.assertEqual(result, constance.config.TERMS_OF_SERVICE_URL)
+
+    @override_config(MFA_ENABLED=True)
+    def test_mfa_value_globally_enabled(self):
+        self.client.login(username='someuser', password='someuser')
+        response = self.client.get(self.url, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertTrue(response.data['mfa_enabled'])
+
+    @override_config(MFA_ENABLED=False)
+    def test_mfa_value_globally_disabled(self):
+        self.client.login(username='someuser', password='someuser')
+        response = self.client.get(self.url, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertFalse(response.data['mfa_enabled'])
+
+    @override_config(MFA_ENABLED=True)
+    def test_mfa_value_globally_on_w_per_user_activations(self):
+        someuser = User.objects.get(username='someuser')
+
+        # Activate MFA for someuser
+        get_mfa_model().objects.create(
+            user=someuser,
+            secret='dummy_mfa_secret',
+            name='app',
+            is_primary=True,
+            is_active=True,
+            _backup_codes='dummy_encoded_codes',
+        )
+        # Enable MFA only for someuser
+        KoboMFAPerUserActivation.objects.create(user=someuser)
```

Comment:
is actually activating MFA (`get_mfa_model().objects.create()`) necessary for `"mfa_enabled": true` to be returned by the environment endpoint? i was a little confused because i figured that this test was to demonstrate that the back end communicates to the front end whether or not MFA is merely available for a particular user, not actually enabled/activated/whatever word we want to use

Reply:
> is actually activating MFA (get_mfa_model().objects.create()) necessary for "mfa_enabled": true

No. If Constance setting is true and no users have activated MFA, it should return `true`.

Reply:
Should I rename the tests? (any suggestions :P?)

Reply:
i still think i'm misunderstanding something. why not remove all of this?
```python
        get_mfa_model().objects.create(
            user=someuser,
            secret='dummy_mfa_secret',
            name='app',
            is_primary=True,
            is_active=True,
            _backup_codes='dummy_encoded_codes',
        )
```
the name wasn't really a problem for me, except that i separately found the word "activation" confusing. since we're replacing "activation" with "available to" in the model name, something like `test_mfa_per_user_availability_while_globally_enabled` might be good.

Reply:
Oh I see what you mean. 
You are right. We can get rid of it. At the beginning I wanted to do some assertions on `is_active` in this test but as you said , it's covered by `MFASignalTestCase`